### PR TITLE
Rollback hpos store

### PR DESCRIPTION
**Pull request by Paterick**
- Remove holoportAPIStore
- Re-integrate holopport logic into holochainStore

### CODE REVIEW

File: src/stores/useHolochainStore.js [hunk 1]
```diff
@@ -104,21 +106,107 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, is_hpos_served, h
...
+            const params = { cellId, signingKey }    
+            const cap_token = await this.hposHolochainCall({path: 'cap_token', headers: {}, params})
+
+            const signingCredentials = {
+              capSecret: new Uint8Array(listify(cap_token, (_, value) => (Number(value)))),
+              keyPair,
+              signingKey
+            }
+
+            await setSigningCredentials(cellId, signingCredentials)
```
Comment (robbiecarlton) [inline src/stores/useHolochainStore.js]: I'm surprised by this. Don't we get zome call signing for free when using hpos holochain api?

https://github.com/Holo-Host/holo-nixpkgs/blob/69f93c029c571753dd5d87b79f8194cd691550ff/overlays/holo-nixpkgs/hpos-holochain-api/src/index.js#L809C1-L811C46

File: src/stores/useHolochainStore.js [hunk 2]
```diff
@@ -104,21 +106,107 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, is_hpos_served, h
...
+          } catch (e) {
+            console.log(`Error setting signing credentials`, e)
+            reject()
+          }
+
+          resolve()
```
Comment (robbiecarlton) [inline src/stores/useHolochainStore.js]: also, the result of this promise is being assigned to `this.signingCredentials` (line 122), but the promise never returns anything and instead calls various state mutating functions (either `adminWs.authorizeSigningCredentials` or `setSigningCredentials`.
Given that, what's the purpose of `this.signingCredentials` and why does this need to be a promise and not just inline `await`s?

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
Couple of clarifying questions/suggestion

File: src/stores/useHolochainStore.js (hunk 2 above)
Comment (Paterick) [reply src/stores/useHolochainStore.js]: You're correct, I removed it.

File: src/stores/useHolochainStore.js (hunk 1 above)
Comment (Paterick) [reply src/stores/useHolochainStore.js]: You're correct, I removed it.

**Review by robbiecarlton — APPROVED** [review]

**Pull request merged**